include/inviwo/core/properties/isotfproperty.h

```diff
@@ -92,8 +92,26 @@ class IVW_CORE_API IsoTFProperty : public CompositeProperty,
     void setZoomV(double zoomVMin, double zoomVMax);
     const dvec2& getZoomV() const;
 
+    /**
+     * Set the HistogramMode to control how to scale the histogram
+     * The options are:
+     *
+     * * Off Don't show any histograms
+     * * All Make sure all the bars are fully visible
+     * * P99 Make sure 99% of the bars are fully visible
+     * * P95 Make sure 95% of the bars are fully visible
+     * * P90 Make sure 90% of the bars are fully visible
+     * * Log Apply logarithmic scaling to each bar.
+     *
+     */
     IsoTFProperty& setHistogramMode(HistogramMode type);
     HistogramMode getHistogramMode() const;
+    
+    /**
+     * Set the HistogramSelection. The selection determine which of the histograms from the volume
+     * in the optional volume port to show. The selection is a bitset, up to 32 histograms are
+     * supported. By default all available histograms will be shown.
```

Comment:
What does "by default" mean? There is no default argument... Do you mean if one has not changed it since TF was constructed?